capa/v25.1.1/announcement.md

```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v25.1.1 for CAPA is available**. This release updates cluster-aws to v1.1.1 which includes several important patches.
```

Comment:
```suggestion
**Workload cluster release v25.1.1 for CAPA is available**. This release updates cluster-aws to v1.1.1 which allows configuring the node CIDR mask size, toggle the auditd service, and define multiple service account issuers in a certain order (important for migration of Vintage clusters).
```

Reply:
I will never understand why English grammar adds a comma in enumerations. Apart from that: Thank you! ❤️